profiles/testbot.lua

```diff
@@ -19,6 +19,14 @@ function setup()
       use_turn_restrictions         = true
     },
 
+    classes = {[0] = "motorway", [1] = "toll"},
+
+    excludable = {
+        [1] = {["motorway"] = true},
+        [2] = {["toll"] = true},
+        [3] = {["motorway"] = true, ["toll"] = true}
```

Comment:
is `= true` really needed? I think also it is better to use plane tables as
```
    excludable = {
        {'motorway'},
        {'toll'},
        {'motorway', 'toll'}
    },
```

It will require only change `first` -> `second` at 
https://github.com/Project-OSRM/osrm-backend/blob/936539ca8a8810e9cf8ebbbd7c7e788200b87b22/src/extractor/scripting_environment_lua.cpp#L722